src/components/Input.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import Radium from "radium";
+import {styles} from "./styles";
+
+export class Input extends React.Component {
+  render () {
+    return (
+      <input
+        placeholder={this.props.defaultValue}
```

Comment:
I don't understand this semantic. Is it a placeholder? Or a default value? Those kind of have different meanings. A placeholder is text that shows up before the user has typed anything into an input. A default value is an actual value to use for the form/input if the user doesn't enter anything. In some cases, you might show the default value in the placeholder, but it looks like you aren't actually using the default value as a value anywhere, which is why I'm confused.